tests/bluetooth/tester/src/gap.c

```diff
@@ -171,11 +172,24 @@ static void le_security_changed(struct bt_conn *conn, bt_security_t level,
 			    CONTROLLER_INDEX, (uint8_t *) &sec_ev, sizeof(sec_ev));
 		break;
 	case BT_SECURITY_ERR_PIN_OR_KEY_MISSING:
-		memcpy(bond_ev.address, addr->a.val, sizeof(bond_ev.address));
-		bond_ev.address_type = addr->type;
+		bt_conn_get_info(conn, &info);
```

Comment:
Please either store the result of this in a variable and potentially log if it returns an error, or add `(void)` to explicitly discard the return value. 

This comment also applies to `memcpy` in line 185, and `bt_conn_set_security` in line 191

Reply:
memcpy doesn't return any error

Reply:
It doesn't return an error, but it does return a pointer: 
```
void *memcpy(void *_MLIBC_RESTRICT d, const void *_MLIBC_RESTRICT s, size_t n)
```

Regardless of the return type, we should add `(void)` if we explicitly ignore that value.

Reply:
fair enough (although it is not very common in zephyr code)

Reply:
It is very common for e.g. `memset`, but for some reason it hasn't been done that much for e.g. `memcpy`.